### PR TITLE
Refactoring indentations, UI and code style

### PR DESCRIPTION
**Pull request by martinpilch**

### CODE REVIEW

File: BullsEye/Players.swift [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+        self.name = name
+        self.address = address
+    }
+
+    func getPhotos() {
+        // we assume that we downloaded some photos from server and assigned them to photos variable
+        let photos = [Any]()
+
+        if let didGetPhotos = didGetPhotos {
+            didGetPhotos(photos)
```
Comment (martinpilch) [inline BullsEye/Players.swift:28]: Tweaking code style

File: BullsEye/PlayersViewController.swift [hunk 2]
```diff
@@ -1,73 +1,70 @@
...
+        let alert = UIAlertController(title: "Change address", message: "Enter new address for player: \(player.name)", preferredStyle: .alert)
+
+        alert.addTextField { (textField) in
+            textField.accessibilityIdentifier = "addressInput"
+            textField.placeholder = "Enter new address"
+        }
+
+        alert.addAction(UIAlertAction(title: "OK", style: .default, handler: { [weak alert] (_) in
+            let textField = alert?.textFields?[0]
+            player.address.fullAddress = textField?.text ?? ""
```
Comment (martinpilch) [inline BullsEye/PlayersViewController.swift:42]: Removing force unwrap

File: BullsEye/ViewController.swift [hunk 3]
```diff
@@ -1,104 +1,93 @@
...
-      segmentedControl.selectedSegmentIndex = defaultGameStyle
-    } else {
-      gameStyle = .moveSlider
-      defaults.set(0, forKey: "gameStyle")
+    @IBOutlet var targetGuessLabel: UILabel!
+    @IBOutlet var targetGuessField: UITextField!
+    @IBOutlet var roundLabel: UILabel!
+    @IBOutlet var scoreLabel: UILabel!
+    @IBOutlet var slider: UISlider!
+    @IBOutlet var segmentedControl: UISegmentedControl!
```
Comment (martinpilch) [inline BullsEye/ViewController.swift:11]: No longer needed to keep weak reference to IBOutlets

File: BullsEye/PlayersViewController.swift [hunk 4]
```diff
@@ -1,73 +1,70 @@
...
-  private func showAlert(player: Player){
-    let alert = UIAlertController(title: "Change address", message: "Enter new address for player: \(player.name)", preferredStyle: .alert)
-    
-    alert.addTextField { (textField) in
-      textField.accessibilityIdentifier = "addressInput"
-      textField.placeholder = "Enter new address"
+    @IBOutlet var player1Name: UILabel!
+    @IBOutlet var player1Address: UILabel!
+    @IBOutlet var player2Name: UILabel!
+    @IBOutlet var player2Address: UILabel!
```
Comment (martinpilch) [inline BullsEye/PlayersViewController.swift:7]: No longer needed to keep weak reference to IBOutlets

**Review by pitr12 — APPROVED** [review]

**Pull request merged**